admin/package.json

```diff
@@ -8,9 +8,9 @@
   "author": "",
   "dependencies": {
     "bootstrap": "3.3.7",
-    "bootstrap-table": "1.11.2",
+    "opencollective": "*",
```

Comment:
I don't understand what's this fore..., pleas specify the version :-)

Reply:
Good catch, I will pin the version. We need this to install bootstrap-table from source, it displays a message about donation on stdout during npm install.

Reply:
OK, thanks :-)